templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">User: {{ user.username }}</a></li>
```

Comment:
Ensure that `user.id` corresponds to the primary key of the `Driver` model. If `user` is not a `Driver`, this URL might not work as expected.